runironic.sh

```diff
@@ -74,8 +74,8 @@ rm -rf /shared/log/ironic
 
 mkdir -p /shared/log/ironic
 
-/usr/bin/python2 /usr/bin/ironic-conductor --log-file /shared/log/ironic/ironic-conductor.log &
-/usr/bin/python2 /usr/bin/ironic-api --log-file  /shared/log/ironic/ironic-api.log &
+/usr/bin/python /usr/bin/ironic-conductor --log-file /shared/log/ironic/ironic-conductor.log &
+/usr/bin/python /usr/bin/ironic-api --log-file  /shared/log/ironic/ironic-api.log &
```

Comment:
so we shouldn't need to directly invoke with python on the path as PBR generates the script it includes a "#!/usr/bin/python". Ultimately we should want to shift to python3, but I'm guessing this is one of the areas current support in centos is problematic to build the images :\

Reply:
I actually did a test with Python 3.x but it was not very sucessfull :/
and yes, you're right, I should just remove the python binary invocation